docs/source/federated-types/federated-directives.mdx

```diff
@@ -48,6 +65,20 @@ type Product @key(fields: "id") {
 }
 ```
 
+You can apply multiple `@key` directives to a single entity (to specify multiple valid sets of key fields), _if_ your subgraph library supports repeatable directives:
+
+```graphql {1}
+type Product @key(fields: "upc") @key(fields: "sku") {
+  upc: ID!
+  sku: ID!
+  name: String
+}
+```
+
+To check whether your subgraph library supports repeatable directives, see the `@KEY (MULTI)` item in [Federation-compatible subgraph implementations](../supported-subgraphs/).
+
+> Although it's valid to apply `@key` to an interface definition, this currently has no effect in Apollo Federation.
```

Comment:
I thought that it would be inherited by the implementing classes? so if you add `@key` to the interface then all implementing classes will have it applied automatically

Reply:
Ooh maybe in Fed 2? Last I knew in Fed 1 this did nothing but was valid syntax. I could be out of the loop though

Reply:
Okay, looks like Fed 2 composition currently straight-up _rejects_ `@key` on interfaces despite being valid syntax, so I'll update this accordingly